LICENSES/vendor/github.com/yujunz/go-getter/LICENSE

```diff
@@ -0,0 +1,358 @@
+= vendor/github.com/yujunz/go-getter licensed under: =
+
+Mozilla Public License, version 2.0
```

Comment:
besides the MPL licensed hashicorp deps, this one too.

Reply:
That is _extremely_ disappointing.  Is there any wiggle room here?

Say I get rid of those deps before the 1.21 code freeze on 1.21; can we get this PR in then?

Reply:
I'm also curious why a fork over https://github.com/hashicorp/go-getter ?

Reply:
It's gone now.  @soltysh - there's some history in kubernetes-sigs/kustomize#3578 if you want to discuss there.